alpha/notify-worker/app/process-notifications.js

```diff
@@ -0,0 +1,34 @@
+var logger = require('./bunyan-logger')
+var client = require('./db-client')
+var mailer = require('./mailer')
+var Promise = require('bluebird')
+
+const EMAIL_CONTACT_METHOD = 'email'
+
+exports.run = function () {
```

Comment:
should have basic unit test covering that it calls to processNotification for all pending notifications